appveyor.yml

```diff
@@ -236,11 +236,24 @@ after_build:
 
 test_script:
 - ps: |
-    & ctest -C $env:CONFIGURATION --output-on-failure
+    & ctest -C $env:CONFIGURATION -O "Testing/Temporary/TestOutput.txt" --output-on-failure
     if ($LastExitCode -ne 0) {
         throw "Exec: $ErrorMessage"
     }
 
+on_finish:
+- ps : |
+    $wc = New-Object 'System.Net.WebClient'
+    $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\test\System\SystemUnitTests.xml))

```

Comment:
I think that you need to change the directory of output (with `TEST_OUTPUT`) and use a loop in that ps script to upload all the xml files.